src/projects.h

```diff
@@ -583,6 +583,7 @@ enum deprecated_constants_for_now_dropped_analytical_factors {
 #define PJD_ERR_ELLIPSOIDAL_UNSUPPORTED -56
 #define PJD_ERR_TOO_MANY_INITS          -57
 #define PJD_ERR_INVALID_ARG             -58
+#define PJD_ERR_INCONSISTENT_UNIT       -59
```

Comment:
You should add a string representation in `src/pj_strerrno.c` as well

Reply:
Good point. I've added a comment at the end of the enumeration to update pj_strerrno.c AND pj_transform.c as well (and I've sanitized the access to the transient_error array in it)